client/src/components/Public/Map/ClusterMarker/ClusterMarker.js

```diff
@@ -22,6 +22,7 @@ type ClusterMarkerProps = {
   cluster: MapCluster,
   onClick: () => void,
   openedAddressId: number,
+  MarkerLabel: boolean | number,
```

Comment:
:thinking: Honestly, I really don't like this confusing type. I'm not sure how to make the API of this component better, maybe we could split this in two props (`isLabel` and `numberOfPoints`). What do you think?

Reply:
we could but the number of points is already in cluster, all information is in cluster prop, but i guess there is no harm in duplicating. I find it also confusing, but less redundant...

Reply:
well if we do not plan to use the names in the label we can actually make it only use label

Reply:
done

Reply:
So, since it isen't used anymore, could you please remove that form propType? Thanks :slightly_smiling_face: